nautobot/extras/tests/test_views.py

```diff
@@ -921,9 +921,9 @@ def setUpTestData(cls):
         )
 
         secrets = (
-            Secret.objects.create(name="secret 1", provider="text-file", parameters={"path": "/tmp"}),
-            Secret.objects.create(name="secret 2", provider="text-file", parameters={"path": "/tmp"}),
-            Secret.objects.create(name="secret 3", provider="text-file", parameters={"path": "/tmp"}),
+            Secret.objects.create(name="secret 1", provider="text-file", parameters={"path": "/tmp"}),  # noqa: S108  # hardcoded-temp-file
```

Comment:
why not `# noqa: S108  # hardcoded-temp-file -- false positive` like in `nautobot/extras/tests/test_api.py`